### DISCUSSION TITLE
之后会支持QQ频道吗？

### DISCUSSION THREAD

**Discussion by moelody** [Tsuk1ko/CQ-picfinder-robot#273] [我有想法]
手机上的QQ频道现在基本可用了，扩展支持到QQ频道是否可行

**Comment by Tsuk1ko** [comment]
会支持

**Comment by Tsuk1ko** [comment]
v2.33.0 起已支持频道

**Comment by Annabel0530** [comment]
发现QQ频道不能搞颜色   涉嫌违规的图片发出去后  如果检测到了都会被自动删除

**Reply by Tsuk1ko** [reply]
Q群也一样，只不过如果在检测出违规之前收到了就留存在消息记录里，后来的人看漫游消息是没有的

频道的话我没试过，删除会影响已经接收到的？

**Comment by Annabel0530** [comment]
> Q群也一样，只不过如果在检测出违规之前收到了就留存在消息记录里，后来的人看漫游消息是没有的
> 
> 频道的话我没试过，删除会影响已经接收到的？

收不到  试了大半个小时 大概有几种情况
1.发出去几秒后 被删除 自己本地窗口的图也没保留
2.跟QQ群一样 发出去保留在自己窗口 但是别人也收不到 同一个号不同客户端也看不到
3.跟第一种差不多  但是BOT能收到 也有回复搜索结果 但是都会被删除 PC端会保留 但是重启后还是会消失
等等
有些图片二次编辑、二次截图之后都是一样的结果 不同设备客户端也会有不一样的情况
还是得多观察一阵时间 甚至有可能得禁止频道使用R18

**Reply by Tsuk1ko** [reply]
感觉是因为是公开的场合所以管控力度会大很多😂
